website/src/react-native/showcase.js

```diff
@@ -111,6 +111,12 @@ var featured = [
     author: 'Townske PTY LTD',
   },
   {
+    name: 'Caratlane',
+    icon: 'https://lh3.googleusercontent.com/wEN-Vvpbnw_n89dbXPxWkNnXB7sALKBKvpX_hbzrWbuC4tFi5tVkWHq8k5TAvdbf5UQ=w300-rw',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.caratlane.android&hl=en',
```

Comment:
you need to use `link` key if it's just one store. `linkPlayStore` is used when you have ios and android both